### ISSUE TITLE
Angular universal

### ISSUE THREAD

**Issue by ihor-zinchenko** [robisim74/angular-l10n#306]
Hi! I'm using angular universal and it build HTML for my app but before translations loading I see 
```
'some.title.or.something.else'
```
How I can load translations immediately and after that build HTML? 

Or for angular universal I should use another way to connect translations?

**Comment by robisim74** [comment]
Hi,

- what version of the library are you using?
- are you initializing the library using the `APP_INITIALIZER`?
- are you using absolute urls of translation files server-side?

For the previous version of the library, I wrote a simple guide: https://github.com/robisim74/angular-l10n/blob/angular_v8/docs/quick-start.md#server-side-rendering-ssr

**Comment by ihor-zinchenko** [comment]
@robisim74 
1. I using the latest version
2. Yep
```
{
      provide: APP_INITIALIZER,
      useFactory: initL10n,
      deps: [L10nLoader],
      multi: true
    },
```
3. No, but I'm solve my issue by moving from JSON to .ts files

But now I have a new problem. The server always returns only first language for the user, if I go to example.com/en, I've got English version, but if I go to example.com/ru after that, I've got English version to

I'll try your example maybe it's fixing my issue, thx!
